specifyweb/backend/bulk_copy/bulk_delete_task.py

```
@@ -0,0 +1,139 @@
+            'type': 'bulk-delete-started',
+            'table': table_name,
+            'count': len(ids) if ids else 'entire query',
+            'taskid': self.request.id,
+        }),
+    )
+
+    try:
+        do_bulk_delete(collection, agent, table_name, ids, spquery, progress)
+
+    except Exception:
+        # 2a) Task failed – notify the user and re-raise
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)